File: service/src/main/java/no/nav/melosys/service/dokument/brev/BrevDataVideresend.java
package no.nav.melosys.service.dokument.brev;

import no.nav.melosys.domain.UtenlandskMyndighet;

public class BrevDataVideresend extends BrevData {
    public String bostedsland;
    public UtenlandskMyndighet trygdemyndighet;

    public BrevDataVideresend(BrevbestillingDto brevbestillingDto, String saksbehandler) {
        super(brevbestillingDto, saksbehandler);
    }
}

File: service/src/main/kotlin/no/nav/melosys/service/dokument/brev/BrevDataVideresend.kt
package no.nav.melosys.service.dokument.brev

import no.nav.melosys.domain.UtenlandskMyndighet

class BrevDataVideresend(
    brevbestillingDto: BrevbestillingDto,
    saksbehandler: String
) : BrevData(brevbestillingDto, saksbehandler) {
    var bostedsland: String? = null
    var trygdemyndighet: UtenlandskMyndighet? = null
}
